app/views/payments/openPayments.scala.html

```diff
@@ -54,19 +54,17 @@ <h1 class="heading-xlarge">@messages("openPayments.heading")</h1>
             </div>
             @if(appConfig.features.allowPayments()) {
               <div class="column-one-third form-hint">
-                  <button class="button--link-style float--right"
-                          type="submit" id="pay-now-@{count + 1}"
-                          data-journey-click="payment:make-payment:open-payments">
+                  <a data-journey-click="payment:make-payment:open-payments"
+                     href="@controllers.routes.MakePaymentController.makePayment((payment.amount * 100).toInt, payment.end.getMonthValue, payment.end.getYear).url" class="float--right">
+                      <span aria-hidden="true">@messages("openPayments.makePayment")</span>
```

Comment:
Can you remove the `aria-hidden="true"`, please? The screen reader is saying all the hidden text to do with the amount and period but not the 'Pay now'